staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/redis/client.go

```diff
@@ -1,35 +1,124 @@
 package redis
 
 import (
+	"context"
+	"encoding/json"
+	"github.com/go-redis/redis/v8"
 	"github.com/kubeedge/Template/pkg/common"
-	"github.com/kubeedge/Template/pkg/global"
+	"k8s.io/klog/v2"
+	"os"
+	"strconv"
+)
+
+var (
+	RedisCli *redis.Client
 )
 
 type DataBaseConfig struct {
+	Config *ConfigData
+}
+
+type ConfigData struct {
```

Comment:
change configdata to redisClientConfig as what we show in device_instance.yaml may be better